src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -222,7 +223,35 @@ private function publishUpdate($object, array $options, string $type): void
             $data = $options['data'] ?? $this->serializer->serialize($object, key($this->formats), $context);
         }
 
-        $updates = array_merge([$this->buildUpdate($iri, $data, $options)], $this->getGraphQlSubscriptionUpdates($object, $options, $type));
+        if ($options['owners_topic'] ?? false) {
+            if (null === $this->expressionLanguage) {
+                throw new RuntimeException('The Expression Language component is not installed. Try running "composer require symfony/expression-language".');
+            }
+
+            $topics = (array) $iri;
+
+            $owners = $this->expressionLanguage->evaluate($options['owners_topic'], ['object' => $object]);
+            $owners = empty($owners) || \is_array($owners) ? $owners : [$owners];
+
+            if (!empty($owners)) {
+                foreach ($owners as $owner) {
+                    $query_topics = '?';
+
+                    foreach ((array) $iri as $topic) {
+                        $query_topics .= \strlen($query_topics) > 1 ? '&' : '';
+                        $query_topics .= sprintf('topic=%s', urlencode($topic));
```

Comment:
Could you use `http_build_query` instead?

Reply:
I was looking for something like this, but I made the change and it was not possible to use it. It turns out that the function receives an array of key / value, making it impossible to use the same key more than once. It is possible to put a prefix, leaving something like `topic_0`, `topic_1`, but I modified the subscribe to suit the pattern and it didn’t work, so I did `https://example.com/{?topic_[0-9]+}`.

So I don't know if Mercure is capable of interpreting the regex, or if it expects anything other than the syntax I know.